src/main/java/com/google/devtools/build/lib/bazel/bzlmod/BazelDepGraphFunction.java

```diff
@@ -249,6 +264,27 @@ static BzlmodFlagsAndEnvVars getFlagsAndEnvVars(Environment env) throws Interrup
     return extensionUsagesTableBuilder.buildOrThrow();
   }
 
+  private static ImmutableBiMap<RepositoryName, ModuleKey> computeCanonicalRepoNameLookup(
+      ImmutableMap<ModuleKey, Module> depGraph) {
+    // Find those modules of which there is only a single version in the dep graph. Currently, the
```

Comment:
s/those modules of which there is/modules with/

may make it a bit easier to read